src/ARKit/ARSkeleton.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+
+using Foundation;
+using ObjCRuntime;
+
+namespace ARKit {
+	public partial class ARSkeleton {
+
+		[iOS (14,0)]
+		[DllImport (Constants.ARKitLibrary)]
+		static extern IntPtr /* NSString */ ARSkeletonJointNameForRecognizedPointKey (/* NSString */ IntPtr recognizedPointKey);
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[iOS (14,0)]
+		public static NSString CreateJointName (NSString recognizedPointKey) {
+			if (recognizedPointKey == null)
+				throw new ArgumentNullException (nameof (recognizedPointKey));
+			var newNamePtr = ARSkeletonJointNameForRecognizedPointKey (recognizedPointKey.Handle);
+			return (newNamePtr == IntPtr.Zero) ? null : new NSString (newNamePtr);
```

Comment:
This makes sure we don't create multiple NSStrings with the same native handle:

```suggestion
			return (NSString) Runtime.GetNSObject (ARSkeletonJointNameForRecognizedPointKey (recognizedPointKey.Handle));
```